From 4520e7813dcbca57ff19cba68085f5d8adf4785e Mon Sep 17 00:00:00 2001
From: Carl George <carl@george.computer>
Date: Wed, 31 Aug 2022 18:00:24 -0500
Subject: [PATCH] fix: Allow installation on GNOME 43

I was able to set org.gnome.shell.disable-extension-version-validation
to true and install the extension on GNOME shell 43 beta.  Everything
appears to work as expected.
---
 metadata.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index c80f61d3..c92b3f21 100644
--- a/metadata.json
+++ b/metadata.json
@@ -8,6 +8,7 @@
         "3.36",
         "40",
         "41",
-        "42"
+        "42",
+        "43"
     ]
 }
